File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.java
class Solution {
    public int getMinDistance(int[] nums, int target, int start) {
        int mindis = Integer.MAX_VALUE;
        for(int i=0;i<nums.length;i++){
            if(nums[i]==target){
                int dis = Math.abs(i-start);
                if(dis < mindis){
                    mindis = dis;  
                }
            }
        }
        return mindis;
        
    }
}

File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.cpp
class Solution {
public:
    int getMinDistance(vector<int>& nums, int target, int start) {
        int minDist = nums.size();  // set to max possible distance
        for (int i = 0; i < nums.size(); i++) {
            if (nums[i] == target) {
                int dist = abs(i - start);
                if (dist < minDist) {
                    minDist = dist;
                }
            }
        }
        return minDist;
    }
};
